tutorials/beaconkit.md

```diff
@@ -0,0 +1,113 @@
+# Building a Sovereign Rollup with BeaconKit and Rollkit
+
+![beaconkit](https://camo.githubusercontent.com/8aaae79e171969a2a9c950582d512cd1e3746e67d3aea6410afc04e9b6cb8055/68747470733a2f2f7265732e636c6f7564696e6172792e636f6d2f6475763067343032792f696d6167652f75706c6f61642f76313731383033343331322f426561636f6e4b697442616e6e65722e706e67)
+
+## Introduction
+
+This tutorial guides developers through the process of building and running a sovereign rollup using BeaconKit and Rollkit. BeaconKit is a modular framework for building EVM-based consensus clients, while Rollkit is a modular framework for building sovereign rollups. By combining these tools, you can create your own customizable rollup chain using the Ethereum Virtual Machine (EVM).
+
+## Prerequisites
+
+Before starting, ensure you have the following software installed on your machine:
+
+* [Rollkit](https://rollkit.dev/): A modular framework for sovereign rollups
+* [Docker](https://www.docker.com/): For running containerized applications
+* [Go](https://golang.org/) (version >= 1.21.0): The programming language used by BeaconKit
+* [jq](https://stedolan.github.io/jq/): A lightweight command-line JSON processor
```

Comment:
Isn't this also in some install script?

Reply:
it is, but shouldn't we mention it anyway?